src/error.rs

```diff
@@ -85,7 +85,7 @@ impl Error {
     }
 
     // backcompat shim
-    #[cfg_attr(feature = "cargo-clippy", allow(not_unsafe_ptr_arg_deref))]
+    #[cfg_attr(feature = "cargo-clippy", allow(clippy::not_unsafe_ptr_arg_deref))]
```

Comment:
AFAIU this can be simply `#[clippy::blablabla]` now without the `cfg_attr` stuff?

Reply:
IMHO 1.31.0 still don't support it, will check at evening.
Thanks for reminder